CMakeLists.txt

```diff
@@ -1,102 +1,105 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(laser_assembler)
 
-##############################################################################
-# Find dependencies
-##############################################################################
-
-set(THIS_PACKAGE_ROS_DEPS 
-tf sensor_msgs message_filters roscpp laser_geometry filters)
-
-find_package(catkin REQUIRED COMPONENTS 
-  ${THIS_PACKAGE_ROS_DEPS} 
-  message_generation)
-find_package(Boost REQUIRED COMPONENTS system signals)
-include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
-
-##############################################################################
-# Build service definitions
-##############################################################################
-add_service_files(FILES
-AssembleScans.srv
-AssembleScans2.srv)
-
-generate_messages(DEPENDENCIES sensor_msgs std_msgs)
-
-##############################################################################
-# Define package
-##############################################################################
-
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS ${THIS_PACKAGE_ROS_DEPS} message_runtime
-  DEPENDS
-  )
-
-##############################################################################
-# Build
-##############################################################################
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+endif()
 
-add_executable(laser_scan_assembler_srv src/laser_scan_assembler_srv.cpp)
-target_link_libraries(laser_scan_assembler_srv ${catkin_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(laser_scan_assembler_srv ${PROJECT_NAME}_gencpp)
+set( filters_FOUND 1 )
```

Comment:
Is this variable used somewhere? How do you know filters was found?

Reply:
It was workaround added previously, because find_package(filters) was giving some errors,
Updating filters package cmake has solved this issue so removed it in next commit.